### ISSUE TITLE
Missing alt texts on search result list icons

### ISSUE THREAD

**Issue by osma** [NatLibFi/Skosmos#1059]
The WebAIM checker (#1043) complains about missing alt texts on the icons shown on the search result lists.
We should use the labels for the corresponding properties as alt texts.

**Comment by osma** [comment]
This was only relevant for Skosmos 2.x. In Skosmos 3, the icons are implemented in a different way (Font Awesome symbols) with proper (hopefully) title attributes. Closing this issue.
